### PR TITLE
[java/en] Enum Type

### PR DESCRIPTION
**Pull request by raphaelbn**
Short overview about enum type.

### CODE REVIEW

File: java.html.markdown [hunk 1]
```diff
@@ -670,6 +671,68 @@ public abstract class Mammal()
...
+                        
+            default:
+                System.out.println("Midweek days are so-so.");
+                break;
+        }
+    }
+    
+    public static void main(String[] args) {
+        EnumTest firstDay = new EnumTest(Day.MONDAY);
+        firstDay.tellItLikeItIs();
```
Comment (levibostian) [inline java.html.markdown]: Instead of having the outputs below, can you put them here inline? 
`firstDay.tellItLikeItIs(); // => Mondays are bad.`

**Comment by raphaelbn** [comment]
Hi @levibostian,

Sure, I've already made the commit.

:+1:

**Comment by raphaelbn** [comment]
Is it ok now, @levibostian ?

**Comment by levibostian** [comment]
Thanks! Now can you merge in master please?

**Comment by raphaelbn** [comment]
Hi @levibostian , I guess I don't have permission to merge..

**Comment by raphaelbn** [comment]
@levibostian, I updated the code and it's not in conflit anymore. 
Can you merge, please?

**Comment by geoffliu** [comment]
Wait wait, one comment: The example given is not really in Java style. Enums in Java can have methods on them, and in the example you gave, the `tellItLikeItIs` method is better in the enum class itself, so you can do 

    Days.MONDAY.tellItLikeItIs()

I would rather show that as an example, and if you really want to show how enums are used in other classes, make up a use that is not intrinsic to the enum class (and therefore belongs in the enum class). I'm thinking something like a separate `Person` class that has `isWorkingOn(Day day)`

**Comment by raphaelbn** [comment]
It's only a little example how basic enum works. I know enum can have methods and be better than I've demonstrated.
I said:
"
// Enum types are much more powerful than we show above. 
// The enum body can include methods and other fields.
// You can se more at https://docs.oracle.com/javase/tutorial/java/javaOO/enum.html
"

And this example is in the oracle oficial doc:
https://docs.oracle.com/javase/tutorial/java/javaOO/enum.html

**Comment by raphaelbn** [comment]
@geoffliu

**Comment by geoffliu** [comment]
Still rather have a slightly different example, but I won't press the point. :smile: 

Could you bring it up to date with master, and I'll merge? Thanks!

**Comment by raphaelbn** [comment]
you're welcome @geoffliu :smile:

It's already done.

Thanks :+1:

**Pull request merged**